packages/integration-tests/suites/replay/multiple-pages/test.ts-snapshots/seg-6-snap-incremental

```diff
@@ -40,5 +29,16 @@
     "id": 15,
     "x": 157,
     "y": 90
+  },
```

Comment:
I have no idea why this is suddenly ordered differently... It seems to be consistently so now. Not sure @cmanallen  / @billyvg if you know, does this make any difference, or is the order of records in this incremental snapshot array not important? I don't really see timestamps here, but not sure...